src/gasman.c

```diff
@@ -967,6 +967,8 @@ TNumAllocFuncBags       AllocFuncBags;
 
 TNumStackFuncBags       StackFuncBags;
 
+TNumExtraMarkFuncBags   ExtraMarkFuncBags;
```

Comment:
Hum... but nothing ever sets this to a non-zero value; and no header mentions this.

So code wanting to set it would have to include `extern TNumExtraMarkFuncBags ExtraMarkFuncBags;` ?